tests/acceptance/features/apiSpacesShares/copySpaces.feature

```diff
@@ -161,20 +161,23 @@ Feature: copy file
       | shareType       | user         |
       | permissionsRole | <space-role> |
     And user "Brian" has sent the following resource share invitation:
-      | resource        | testshare |
-      | space           | Personal  |
-      | sharee          | Alice     |
-      | shareType       | user      |
-      | permissionsRole | Viewer    |
+      | resource        | testshare          |
+      | space           | Personal           |
+      | sharee          | Alice              |
+      | shareType       | user               |
+      | permissionsRole | <permissions-role> |
     When user "Alice" copies file "/project.txt" from space "Project" to "/testshare/project.txt" inside space "Shares" using the WebDAV API
```

Comment:
IMO, we don't have to test with secure-view here. we want to test the reverse (from Secure-view share to Project space)